examples/sample/release.yaml

```diff
@@ -14,7 +14,8 @@ spec:
 #       namespace: default
 #     url: "https://charts.bitnami.com/bitnami/wordpress-9.3.19.tgz"
     namespace: wordpress
-#    wait: true
+#   skipCreateNamespace: true
+    wait: true
```

Comment:
I'd prefer to keep wait disabled in the example, if there is no special reason.

Reply:
/ping @srueg

Reply:
The reason I added it was so that the e2e test actually waits for the release to be installed. 
But I agree, it probably doesn't make for a good example.